app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,20 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>MediaRanker</title>
+  <title>Media Ranker</title>
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
 </head>
 <body>
-
-<%= yield %>
-
+  <div class="container">
+    <div class="row">
+      <div class="page-header">
+        <h1><%= link_to "Media Ranker", root_path %>
+        <small>Ranking the Best of Everything</small></h1>
+      </div>
+    </div>
+    <%= yield %>
```

Comment:
Just to make it easier to see, I like to leave a blank line above/below `<%= yield %>`.